### PR TITLE
Help with fees and HWF reference screen

### PR DESCRIPTION
**Pull request by hcarslaw**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/application-within-proceedings/help-with-fees/content.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import { APPLICATION_WITHIN_PROCEEDINGS_GUIDANCE } from '../../../steps/urls';
+import { generateCancelLink, getApplicationDetails } from '../utils';
+
+export const en = {
+  title: 'Help with fees',
+  applicationCost: 'The cost of this application is ',
+  checkGuidance:
+    'You can check the help with fees guidance on <a href="https://www.gov.uk/government/publications/apply-for-help-with-court-and-tribunal-fees/how-to-apply-for-help-with-fees-ex160a" class="govuk-link" rel="external" target="_blank">GOV.UK (opens in a new tab)</a> to find out if you are eligible for support.',
+  usingHelpWithFees: 'Will you be using help with fees to pay for this application?',
+  one: 'Yes',
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/help-with-fees/content.ts:15]: use yes and no as keys

File: src/main/steps/application-within-proceedings/utils.ts [hunk 2]
```diff
@@ -358,3 +361,18 @@ export const getApplicationDetails = (
 
   return appDetails;
 };
+
+export const generateCancelLink = (
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/utils.ts:365]: May I know the rationale behind this method?

File: src/main/steps/applicationWithinProceedingsSequence.ts [hunk 3]
```diff
@@ -1,10 +1,39 @@
...
+  DASHBOARD_URL,
+  PageLink,
+} from './urls';
 
 export const applicationWithinProceedingsSequence: Step[] = [
   {
     url: APPLICATION_WITHIN_PROCEEDINGS_GUIDANCE,
     showInSection: Sections.ApplicationWithinProceedings,
-    getNextStep: () => APPLICATION_WITHIN_PROCEEDINGS_GUIDANCE,
+    getNextStep: (_userCase, req) =>
```
Comment (vivek-sekhar) [inline src/main/steps/applicationWithinProceedingsSequence.ts:17]: Remove this temp navigation.

File: src/main/steps/application-within-proceedings/help-with-fees/content.ts (hunk 1 above)
Comment (hcarslaw) [reply src/main/steps/application-within-proceedings/help-with-fees/content.ts]: Changed keys to yes and no

File: src/main/steps/application-within-proceedings/utils.ts (hunk 2 above)
Comment (hcarslaw) [reply src/main/steps/application-within-proceedings/utils.ts:365]: I'm using this to create the cancel links with the correct application type and reason in the url

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**